README.md

```diff
@@ -1,4 +1,23 @@
 Fences
 ======
 
-A module adding fence blocks
+This module adds a simple feature, a fence block.
+
+Each fence block can be connected to up to four other blocks, with which it is lying side to side.
+Currently, however, the module is a little bit bugged, which means, that the
+fences will not get connected correctly the moment they are placed. However,
+placing an arbitrary block right next to a wrongly placed fence and then mining
```

Comment:
"Currently however, the module is a little bit bugged. This means that the fences do not connect correctly when they are placed. Forcing an update by placing a breaking a block adjacent to the fences will cause them to align together."

Reply:
If you are placing multiple comma's in a rather long sentence then it's probably better to break it up. It'll mean you have less "filler" words, things like the "which means" "however" and the like. This with repeating words/phrases makes it flow badly.